docs/style.md

```diff
@@ -0,0 +1,107 @@
+# Style guide
+
+Please adhere to the following style guide when writing code for the RISCy Kernel.
+
+## DRY
+
+**Don't Repeat Yourself.** <br>
+**Don't Repeat Yourself.** <br>
+**Don't Repeat Yourself.**
+
+Its easy to copy-paste. It's also easy to duplicate bugs that way. Just define a `static` functions, even if it is five lines long and takes four arguments. 
+The compiler is smart and will inline it appropriately.
```

Comment:
@Nino you're gonna hate what I did to your `syscon` code then. (Admittedly there I know it was bad, I just was lazy and left a note)

Reply:
I'm @egelja... you just pinged some random person

Reply:
...

whoops

Reply:
I'll clean up the code